src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -536,7 +536,7 @@ def configure_worker_command(self, *, config: DeadlineWorkerConfiguration) -> st
                 + f"--region {config.region} "
                 + f"--user {config.agent_user} "
                 + f"{'--allow-shutdown ' if config.allow_shutdown else ''}"
-                + "--start"
+                + f"{'--start ' if config.start_service else ''}"
```

Comment:
Have you tested on the e2e test to make sure worker spin up as expected?

Reply:
Yep, used `hatch run cross-os-e2e-test`